components/brave_extension/extension/brave_extension/background/reducers/shieldsPanelReducer.ts

```diff
@@ -272,11 +272,13 @@ export default function shieldsPanelReducer (
               }
             )
             chrome.tabs.executeScript(tabData.id, {
-              code: 'try { window.sessionStorage.clear(); } catch(e) {}'})
+              code: 'try { window.sessionStorage.clear(); } catch(e) {}'
```

Comment:
did you mean to include changes in this file in this PR? If so, please split to a different commit.

Reply:
This file had a trailing comma that tslint was complaining about. After i removed and saved prettier formatted these 2 lines.

Reply:
Yeah ok in general let's try not to combine unrelated changes to files, especially in otherwise non-related files, in commits. Among the disadvantages are that it makes uplifts harder and file commit history confusing.